addons/web/static/tests/core/hotkeys/hotkey_service_tests.js

```diff
@@ -15,6 +16,18 @@ const serviceRegistry = registry.category("services");
 let env;
 let target;
 
+function displayHotkeysOverlay() {
+    if (isMacOS()) {
+        window.dispatchEvent(new KeyboardEvent("keydown", { key: "alt", ctrlKey: true }));
+    } else {
+        window.dispatchEvent(new KeyboardEvent("keydown", { key: "alt", altKey: true }));
+    }
+}
+
+function getOverlayHotkeys(comp) {
+    return [...comp.el.querySelectorAll(".o_web_hotkey_overlay")].map((el) => el.innerText);
+}
```

Comment:
No need to make a "global" function as you use it only in one test.
```suggestion
```
See below comment.